package.json

```diff
@@ -1,17 +1,33 @@
 {
-  "name": "",
+  "name": "reviews_module",
   "version": "",
   "description": "",
+  "main": "index.js",
   "author": "",
   "license": "",
```

Comment:
optional: fill in version 1.0.0, description of the app, author name, and license: UNLICENSED